### PR TITLE
Added the ability to pass service name to `invoke cli`.

### PR DESCRIPTION
**Pull request by gsnider2195**
Tested in app-dev-example https://github.com/nautobot/nautobot-app-dev-example/pull/53

### CODE REVIEW

File: nautobot-app/{{ cookiecutter.project_slug }}/tasks.py [hunk 1]
```diff
@@ -413,9 +413,9 @@ def shell_plus(context):
 
 
 @task
-def cli(context):
+def cli(context, service="nautobot"):
     """Launch a bash shell inside the Nautobot container."""
```
Comment (smk4664) [inline nautobot-app/{{ cookiecutter.project_slug }}/tasks.py:417]:
```suggestion
    """Launch a bash shell inside the container."""
```

File: nautobot-app/{{ cookiecutter.project_slug }}/tasks.py [hunk 2]
```diff
@@ -413,9 +413,9 @@ def shell_plus(context):
 
 
 @task
```
Comment (smk4664) [inline nautobot-app/{{ cookiecutter.project_slug }}/tasks.py:415]:
```suggestion
@task(
    help={
        "service": "Docker compose service name to launch cli in (default: nautobot).",
    },
)
```

**Review by smk4664 — APPROVED** [review]
Thanks! Just left two suggestions though.

File: nautobot-app/{{ cookiecutter.project_slug }}/tasks.py [hunk 3]
```diff
@@ -412,10 +412,14 @@ def shell_plus(context):
...
 
 
-@task
-def cli(context):
-    """Launch a bash shell inside the Nautobot container."""
-    run_command(context, "bash")
+@task(
+    help={
+        "service": "Docker compose service name to launch cli in (default: nautobot).",
+    },
```
Comment (gsnider2195) [inline nautobot-app/{{ cookiecutter.project_slug }}/tasks.py:418]:
```suggestion
    }
```

**Review by smk4664 — APPROVED** [review]

**Pull request merged**